admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -145,6 +144,36 @@ this:
 
 .. _db-postgresql-label:
 
+Configure MySQL for 4-byte Unicode Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For supporting such features as emoji, you have to both enable 4-byte Unicode support in MySQL (instead of the default 3) as well as in ownCloud. 
+To do so:
+
+1. In ``config/config.sample.php`` set ``mysql.utf8mb4`` to true
```

Comment:
@mmattel, I'm just now coming back to this and as I'm reading through the discussion here, the code, and [owncloud/core#27409 (comment)](https://github.com/owncloud/core/issues/27409#issuecomment-287396398) the point about it being for new installations, not upgrades seems incorrect. Can you clarify?